app/src/main/java/org/eyeseetea/malariacare/domain/exception/PushReportException.java

```diff
@@ -0,0 +1,20 @@
+package org.eyeseetea.malariacare.domain.exception;
```

Comment:
@idelcano I think it would be convenient to group the exceptions related to pushreport within the same package, for example Push or another similar name